docs/man/gtkwave.1.md

```diff
@@ -30,7 +30,7 @@ use IBM EDA toolsets.
 
 **-n**,**\--nocli** \<*directory name*\>
 
-:   Use file requester for dumpfile name.
+:   Use file selection dialog for dumpfile or savefile.
```

Comment:
Just a heads up: I noticed that you probably changed this file manually, because it differs in the dot at the end of the line. You can use the `generate_man_pages.sh` script to rebuild the markdown version of the man pages from the files in the `man` directory. But I've also forgotten to regenerate the docs after the recent changes to the man pages, which I'll fix soon.

Reply:
But now that I think about that some more, maybe we shouldn't do that at all and instead regenerate the markdown from the LTS version, which the docs target at the moment. Maybe it wasn't such a good idea to put the docs directory in the same repo as the development version of GTKWave.

Reply:
Oh, that's right. I forgot the docs should reflect the LTS version of GTKWave, not the experimental 3.4 version.